script.js

```diff
@@ -250,3 +252,91 @@ function dissappearParent(e) {
   }
 }
 parentModal.addEventListener('click', dissappearParent);
+
+// storing data to clud local storage done with my partner andrea work.
+
+// const formName = document.getElementById('name1');
+// const email = document.getElementById('text1');
+// const formText = document.getElementById('msg');
+
+// function autoFill() {
+//   const storageObj = {
+//     name: localStorage.setItem('customerName', formName.value),
+//     email: localStorage.setItem('customerEmail', email.value),
+//     customerInfo: localStorage.setItem('customerInfo', formText.value),
+//   };
+//   return storageObj;
+// }
+
+// formName.value = localStorage.getItem('customerName');
+// email.value = localStorage.getItem('customerEmail');
+// formText.value = localStorage.getItem('customerInfo');
+
+// autoFill();
+
+// formName.onchange = autoFill;
+// email.onchange = autoFill;
+// formText.onchange = autoFill;
+
+// Full form validation person project
+
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.style.visibility = 'visible';
+  setTimeout(() => {
+    msg.style.visibility = 'hidden';
+  }, 3000);
+  msg.innerText = message;
+  input.className = type ? '' : 'input-error';
+  return type;
+}
+
+function showError(input, message) {
+  return showMessage(input, message, false);
+}
+
+function showSuccess(input) {
+  return showMessage(input, '', true);
+}
+
+function hasValue(input, message) {
+  if (input.value.trim() === '') {
+    return showError(input, message);
+  }
+  return showSuccess(input);
+}
+
+function validateEmail(input, requiredMsg, invalidMsg) {
+  // check value
+  if (!hasValue(input, requiredMsg)) {
+    return false;
+  }
+
+  // Regegura
+
+  const emailRegex = /^[a-z][a-z0-9_@.]*$/;
+
+  const email = input.value.trim();
+  if (!emailRegex.test(email)) {
+    return showError(input, invalidMsg);
+  }
+  return true;
+}
+
+const form = document.querySelector('#contact-from-validation');
+
+const NAME_REQUIRED = 'Please enter your name';
+const EMAIL_REQUIRED = 'Please enter your email';
+const EMAIL_INVALID = 'Your email address should be in lowercase';
```

Comment:
- [optional] kindly make the error message color to be red so it can easily be noticed